### PR TITLE
Add semantic release

### PR DESCRIPTION
**Pull request by gabor-balla-emarsys**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-    "coverage": "nyc npm test"
+    "coverage": "nyc npm test",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "pre-commit": [
     "code-style"
   ],
+  "release": {
+    "verifyConditions": {
+      "path": "./node_modules/semantic-release/dist/lib/plugin-noop.js"
```
Comment (gergelyke) [inline package.json]: don't do this, it can be run accidentally - they now have travis and codeship verifiers

**Comment by gabor-balla-emarsys** [comment]
We should update this then:
https://github.com/emartech/js-conventions/wiki/Create-NPM-module-with-semantic-release

**Comment by gergelyke** [comment]
go ahead :)

**Comment by gabor-balla-emarsys** [comment]
As discussed, we will use this for now, and update with the codeship verifier later.

**Pull request merged**